cpu/efm32/periph/Kconfig

```diff
@@ -39,14 +39,14 @@ config MODULE_PERIPH_RTC_SERIES1
         help
         RTC driver implementation for EFM32 series 1.
 
-config MODULE_PERIPH_RTT_SERIES0
+config PERIPH_RTT_SERIES0
```

Comment:
I'm not sure why I made all these modules. They do not correspond to the makefile modules. The only think I can think of, is that I wanted to get rid of the manual selection that occurs in the periph Makefile (which is based in the cpu info).

Reply:
It would be better that way indeed, but its missing the Makefile adaptation, would rather have it as out of scope for this one.